brave/brave-core
# Ledger support for Solana signTransaction and signAllTransactions API

Resolves https://github.com/brave/brave-browser/issues/23842

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open Solana app on Ledger.
2. Import an Solana account from Ledger.
3. Make that account your selected account.
4. Visit https://pwgoom.csb.app/ and connect your ledger account (make sure network is correctly set as your selected network)
5. Test Sign and Send Transaction*, SignTransaction*, and SignAllTransactions*, check if every button yields a successful result using ledger.
6. Test Sign Message, should get an operation unsupported error message after click Sign button in our panel UI.
7. Test swap via raydium.io, make sure you set "Allow blind sign" to Yes before testing this step and can disable it after finish testing this.
